File: 백준/Bronze/10809. 알파벳 찾기/알파벳 찾기.java
import java.util.Scanner;

class Main {
		public static void main(String[] args) {
		
			Scanner sc = new Scanner(System.in);

			String test = sc.nextLine();

			char [] arr = test.toCharArray();

			for (int i = 97 ; i < 123 ; i++) {
				if (test.contains("" + (char)i )) {
					
					System.out.print( test.indexOf("" + (char)i)+ " ");
				} else {
					System.out.print(-1 + " ");
				}
			}


	}

}

File: 백준/Bronze/10809. 알파벳 찾기/알파벳 찾기.kt
fun main() {
    var str = readln().toString().toLowerCase().toCharArray()
    var arr = Array(26){'0'}
    var arrCC = Array(26){-1}
    for (i in arr.indices) arr[i] = (97+i).toChar()
    for (i in str.indices) if (arr.contains(str[i]) && arrCC[arr.indexOf(str[i])] == -1)  arrCC[arr.indexOf(str[i])] = i
    for(i in arrCC) print("$i ")
}
